CRM/Upgrade/Incremental/sql/5.68.alpha1.mysql.tpl

```diff
@@ -1 +1,3 @@
 {* file to handle db changes in 5.68.alpha1 during upgrade *}
+
+UPDATE `civicrm_tag` SET `label` = `name`;
```

Comment:
This will overwrite it if somebody does the upgrade, then edits the label, then reruns the upgrade later. Suggest adding `IS NULL OR label=''`.

Also I see you've put a todo lower down about localizable so ok.

Reply:
That's not really a data-loss scenario, but ok.
And yea, `description` wasn't localized either so I added two todos!